InstallPackagesUbuntu.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 # These are the packages required for install on Ubuntu systems.
-sudo apt-get install g++ libqt4-dev libeigen3-dev protobuf-compiler libprotobuf-dev libdc1394-22 libdc1394-22-dev cmake libv4l-0
+sudo apt-get install g++ libqt4-dev libeigen3-dev protobuf-compiler libprotobuf-dev libdc1394-22 libdc1394-22-dev cmake libv4l-0 libopencv-dev
```

Comment:
You should add libopencv-dev to the documentation for the requirements for Basler support. Since it it not required for the core build, it should not go here. Please revert this line.